README.md

```diff
@@ -1,2 +1,2 @@
-New XBlock
+In Video Quiz XBlock
```

Comment:
We should flesh this out. It should describe how to use/configure, including "what string do I have to add to ADVANCED_SETTINGS to enable this?".